pkg/server/apiHandler.go

```diff
@@ -175,67 +113,36 @@ func (h *apiHandler) ping(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
 }
 
-func (h *apiHandler) configPost(w http.ResponseWriter, r *http.Request) {
+func (h *apiHandler) config(w http.ResponseWriter, r *http.Request) {
```

Comment:
There were two function calls: `configPost()` to handle `POST /config`, and `config()` to handle `GET /config`. It looks like these are collapsed into one `config()` now, but it looks like they originally handled the request slightly differently. `GET /config` sent the raw data as is, while `POST /config` pre-processed that.

Why are we merging them into the same? Is one of them deprecated? If so, we need to note that clearly.

Reply:
Return it back for now. Probably, it as point for next PRs.